### ISSUE TITLE
data type buttons Collection phase

### ISSUE THREAD

**Issue by annaasklof** [ScilifelabDataCentre/RDM-guidelines#223]
We also have data type buttons on this page: https://data-guidelines.scilifelab.se/data-life-cycle/collect/. We might want to edit these accordingly to the new style we have on the sharing phase page.

**Comment by YvonneKallberg** [comment]
True, and while doing so also fix added space btw expanded card and next button, as it is now, the next button seems attached to the previous card (check also other pages with buttons, if they need to be fixed)
